src/entity/CommonGame.java

```diff
@@ -0,0 +1,106 @@
+package entity;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class CommonGame implements Game {
+    private int ID;
+    private int score;
+    public String difficulty;
+    public String genre;
+    public int InitialLives;
+    public int max_rounds;
+    public int current_lives;
+    public int rounds_played = 0;
+    private final LocalDateTime createdAt;
+    private LocalDateTime FinishedAt;
```

Comment:
We should probably make all of these public variables private. It would also be a good idea to keep our variable naming consistent and use the Java standard `camelCase` format as opposed to `snake_case` or `PascalCase`.

Reply:
Good point! Thanks Rayce.